### ISSUE TITLE
少了一个参数

### ISSUE THREAD

**Issue by 1607722534** [taoqi98/FedNewsRec#6]
loss = fed_single_update(model,doc_encoder,user_encoder,num,get_user_data,train_uid_table)

**Comment by taoqi98** [comment]
Thank you for your attention on our work. Could provide more details on this issue?

**Comment by 1607722534** [comment]
函数调用要7个参数，你只传了六个

**Comment by taoqi98** [comment]
Thank you for pointing out this issue, I have fixed my code.
